app/main.py

```diff
@@ -1,2 +1,101 @@
+from typing import Any, List
+
+
+class Node:
+    def __init__(self, key: Any, hash_value: int, value: Any) -> None:
+        self.key = key
+        self.hash_value = hash_value
+        self.value = value
+
+
 class Dictionary:
-    pass
+    def __init__(self, initial_capacity: int = 8,
+                 load_factor: float = 0.75) -> None:
+        self.load_factor = load_factor
+        self.capacity = initial_capacity
+        self.size = 0
+        self.hash_table: List[List[Node]] = [[] for _ in range(self.capacity)]
+
+    def _hash(self, key: Any) -> int:
```

Comment:
```suggestion
    def _hash(self, key: Hashable) -> int:
```
please fix everywhere